README.md

```diff
@@ -53,7 +53,7 @@ with this:
 import { rxConnect } from "rx-connect";
 
 @rxConnect(
-    Rx.Observable.timer(0, 1000).timestamp()
```

Comment:
I'm quite sure it's still there :)
http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#instance-method-timestamp

Reply:
That's weird. The RxJS4 to RxJS5 migration guide actually covers this one specifically:

https://github.com/ReactiveX/rxjs/blob/master/MIGRATION.md

<img width="694" alt="screen shot 2017-09-05 at 10 56 28" src="https://user-images.githubusercontent.com/9586897/30050980-0789fd54-9229-11e7-97d8-9772cff50f33.png">

I remember it not working at some point, but perhaps it got re-added? Will try because it makes the example a lot simpler and adds "wow factor" 😄

Reply:
It seems it works but need to map it to a plain object now: https://codesandbox.io/s/vn9qjnqovl

Reply:
@jseminck Hm... I'm thinking maybe we can relax the check here:
https://github.com/bsideup/rx-connect/blob/e586b5e31750a551387f5b700db7ed7ce22e2dd7/src/rxConnect.js#L67-L78

i.e. check if it's function first, if not - do the assignment. WDYT?

Reply:
Yeah, will try it 😄

Reply:
It doesn't really fix the issue here. `.timestamp()` returns an object, just not a plain object. From the docs: Observable<Timestamp<any>>

---

What seems to work is adding this (along with lodash.isobject). But I don't know what other implications it has ...


```js
                    if (isObject(mutation)) {
                        return Object.assign({}, state, {...mutation});
                    }
```

Reply:
sounds good to me :)

Reply:
I'll try it and run my branch on our code, to see if things still work 👍